example/src/Code.re

```diff
@@ -1,4 +1,6 @@
 open Js_of_ocaml;
+open React.Dom.Dsl;
```

Comment:
I missed that on another PR, but any thought on having React.DOM.DSL?

Reply:
@jchavarri @glennsl

Reply:
I have no particular opinion :neutral_face:

Reply:
True, not super strong on those things, just read Ds1 in my head and broke it

Reply:
Another advantage of current api is that if we decide to move it to a file `dom.mli` or `dsl.mli` in the future, the api / module names would not need to change. To avoid changes in uppercase, the files would need to be called `DOM.mli` or `DSL.mli`.